src/signature_pad.js

```diff
@@ -404,7 +406,7 @@ SignaturePad.prototype._toSVG = function () {
 
         path.setAttribute('d', attr);
         path.setAttribute('stroke-width', (widths.end * 2.25).toFixed(3));
-        path.setAttribute('stroke', this.penColor);
+        path.setAttribute('stroke', color);
```

Comment:
If there's no default value for color here (and for drawing dots as well), will it work with the existing exported SVG strings? Won't it set color to `undefined`?